pkgs/top-level/all-packages.nix

```diff
@@ -10136,7 +10136,7 @@ with pkgs;
   };
 
   etcd = etcd_3_5;
-  etcd_3_4 = callPackage ../servers/etcd/3.4.nix { };
+  etcd_3_4 = callPackage ../servers/etcd/3_4 { };
```

Comment:
The underscore (`_`) is on purpose to standardize. 3.5 is changed in another PR.